File: backend/board-service/src/main/java/com/example/boardservice/dto/rabbit/ProjectCreatedEvent.java
package com.example.boardservice.dto.rabbit;

import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Setter
@Getter
public class ProjectCreatedEvent {
    private long projectId;
    private String projectName = "";
    private long creatorId;
    private Instant createdAtUtc = Instant.now();
}

File: backend/dashboard-service/Backend.Dashboards.Api/Dto/ProjectCreatedEvent.cs
namespace Backend.Shared.DTOs;

public class ProjectCreatedEvent
{
    public long ProjectId { get; set; }
    public string ProjectName { get; set; } = string.Empty;
    public long CreatorId { get; set; }
    public DateTime CreatedAtUtc { get; set; } = DateTime.UtcNow;
}